### PR TITLE
Time since fire params

### PR DESCRIPTION
**Pull request by DamianRodziewicz**

### CODE REVIEW

File: R/timeSinceFire.R [hunk 1]
```diff
@@ -35,11 +38,13 @@ timeSinceFireUI <- function(id, rastersNumber) {
...
 #' @param session Shiny server session object
 #' @param rasters Set of rasters to be displayed
+#' @param leafletZoomInit Initial leaflet zoom
+#' @param studyArea Size of study area. Options: "FULL", "EXTRALARGE", "LARGE", "MEDIUM", "NWT", "SMALL"
 #'
 #' @author Mateusz Wyszynski
 #'
 #' @export
-timeSinceFire <- function(input, output, session, rasters) {
+timeSinceFire <- function(input, output, session, rasters, leafletZoomInit, studyArea) {
```
Comment (mateuszwyszynski) [inline R/timeSinceFire.R]: Maybe it's a good idea to add some default values for these two new parameters?

**Review by mateuszwyszynski — CHANGES_REQUESTED** [review]
Just two comments:
- One minor about default values. 
- Second one: I think we should also add slider.Rd file, so the user can access help page of slider module server function.
 
Except that everything works. I've tested changes using the LandWeb app.

File: R/timeSinceFire.R (hunk 1 above)
Comment (DamianRodziewicz) [reply R/timeSinceFire.R]: Sure, done

**Comment by DamianRodziewicz** [comment]
Comments addressed, ptal.

**Review by mateuszwyszynski — APPROVED** [review]

**Comment by mateuszwyszynski** [comment]
LGTM :)

**Pull request merged**